### ISSUE TITLE
Feature request: add support for MLJ

### ISSUE THREAD

**Issue by DilumAluthge** [Nosferican/Econometrics.jl#51]
It would be great to be able to use Econometrics.jl models in MLJ.

In particular, I would really like to see the Econometrics.jl multinomial logistic regression model available in MLJ.

cc: @Nosferican

**Comment by Nosferican** [comment]
Indeed. It is in my plans. I believe it just needs to implement `predict` and register it. I have a few days of annual leave saved up so probably will get to do some work during my next break.

**Comment by DilumAluthge** [comment]
This PR does everything except for the implementation of `predict`: https://github.com/Nosferican/Econometrics.jl/pull/52
